src/app/lib/config/services/afaanoromoo.js

```diff
@@ -49,6 +49,11 @@ export const service = {
       currentPage: 'Fuula kan ammaa',
       skipLinkText: 'Qabiyyeetti darbi',
       relatedContent: 'Odeessa kana irratti dabalata',
+      mediaAssetPage: {
```

Comment:
Do we not know the translations for any services yet or are they all meant to use English?

Reply:
We do not know the translations yet. The translations will come in later

Reply:
We no don't know the translations yet. They are coming later